bin/run_ascat.r

```diff
@@ -10,19 +10,30 @@ if(length(args)<6){
     tumorname = args[5]
     baseDir = args[6]
     gcfile = args[7]
+    gender = args[8]
 }
 
-source(paste(baseDir,"/scripts/ascat.R", sep=""))
+#source(paste(baseDir,"/scripts/ascat.R", sep=""))
```

Comment:
```suggestion
```

Reply:
You could even remove the `ascat.R` file from `bin`

Reply:
For that I guess you would need to be in your own branch (same for the modules stuff).
I'll just remove the file and then commit the change.
You might have to pull your branch if you accepted my suggestion.